app/models/history.rb

```diff
@@ -39,6 +39,15 @@ class History < ApplicationRecord
 
   belongs_to :patient
 
+  after_save :touch_patient
+  before_destroy :touch_patient
```

Comment:
I think this could just be after_create.

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/683/commits/1987337042767c09797d9cb63d699f585d003f30